lib/rubocop/cop/style/return_nil_in_predicate_method_definition.rb

```diff
@@ -52,7 +52,7 @@ class ReturnNilInPredicateMethodDefinition < Base
         include AllowedMethods
         include AllowedPattern
 
-        MSG = 'Use `return false` instead of `%<prefer>s` in the predicate method.'
+        MSG = 'Use `%<prefer>s` instead of `%<current>s` in the predicate method.'
```

Comment:
Perharps "Return ... in predicate methods." instead of "Use ... in the predicate method"?

Reply:
Sure! I've updated this PR.